components/library/objfw/library.p5m

```diff
@@ -0,0 +1,35 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 Jonathan Schleifer
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license LICENSE.QPL license=QPL
+license LICENSE.GPLv3 license=GPLv3
+license LICENSE.GPLv2 license=GPLv2
+
+<transform file -> add pkg.depend.bypass-generate libgcc_s\.so\.1>
```

Comment:
This is very very suspicious!

Reply:
Without this, it complains about not being able to resolve the dependency. Even adding gcc-10-compat-links as a dependency does not help. I was unable to find another solutions to this, as were the people in IRC.